app/src/main/java/de/westnordost/streetcomplete/osm/cycleway/CyclewayParser.kt

```diff
@@ -122,7 +124,14 @@ private fun parseCyclewayForSide(
             if (isSegregated) TRACK else SIDEWALK_EXPLICIT
         }
         "separate" -> SEPARATE
-        "no", "opposite" -> NONE
+        "opposite" -> NONE
+        "no" -> when (isCyclingOkOnSidewalk) {
+            true -> SIDEWALK_OK
+            false -> when (isCyclingDesignatedOnSidewalk) {
+                true -> SIDEWALK_EXPLICIT
+                false -> NONE
+            }
+        }
```

Comment:
I was about to write "uh... that seems it does not work?" until I realized that it is just written in a confusing way. Let me correct that.

```suggestion
        "opposite" -> NONE
        "no" -> when {
            isCyclingOkOnSidewalk -> SIDEWALK_OK
            isCyclingDesignatedOnSidewalk -> SIDEWALK_EXPLICIT
            else -> NONE
        }
```